frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -275,5 +294,82 @@ function CollectionSectionHeading({
     </SidebarHeadingWrapper>
   );
 }
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export default MainNavbarView;
+
+function SidebarOnboardingSection({ initialState }: { initialState: boolean }) {
+  const applicationName = useSelector(getApplicationName);
+
+  return (
+    <Box
+      m={0}
+      bottom={0}
+      pos="fixed"
+      w={NAV_SIDEBAR_WIDTH}
+      bg="bg-white"
+      className={cx({ [CS.borderTop]: !initialState })}
+    >
+      <Box px="md" py="md">
+        {/*eslint-disable-next-line no-unconditional-metabase-links-render -- This link is only temporary. It will be replaced with an internal link to a page. */}
+        <ExternalLink href={getLearnUrl()} className={CS.noDecoration}>
+          {/* TODO: We currently don't have a `selected` state. Will be added in MS2 when we add the onboarding page. */}
+          <PaddedSidebarLink icon="learn">
+            {t`How to use ${applicationName}`}
+          </PaddedSidebarLink>
+        </ExternalLink>
+      </Box>
+      <Box px="xl" pb="md" className={cx({ [CS.borderTop]: initialState })}>
+        {initialState && (
+          <Text
+            fz="sm"
+            my="md"
+            style={{ lineHeight: "1.333" }}
```

Comment:
I was surprised to learn that we're forcing the 1.5 line height on both "md" (14px) and "lg" (17px) font sizes. 
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/ui/components/typography/Text/Text.styled.tsx#L19-L27

But I wasn't sure should I add another theme override for "sm" (12px) or should I use `style` prop here?

Reply:
Mantine components have a prop for line-height, even though it has a not so pretty short name "lh"

Reply:
Wow!
Thank you.

I was searching for it but it didn't occur to me to try `lh`.

Reply:
Yeah, some props have guessable names, some others don't 😆